### ISSUE TITLE
MicroStack documentation claims it runs bug-free

### ISSUE THREAD

**Issue by coreycb** [canonical-web-and-design/ubuntu.com#9002]
This should be reworded. I don't think we can claim any software runs bug-free.

"At time of writing, MicroStack has been assured to run bug-free on Ubuntu 18.04 LTS and Ubuntu 20.04 LTS."

Perhaps can reword this to:

"MicroStack is supported on Ubuntu 18.04 LTS and Ubuntu 20.04 LTS."

**Comment by coreycb** [comment]
That is on this page:
https://ubuntu.com/tutorials/microstack-get-started

**Comment by pmatulis** [comment]
I edited that tutorial to reflect what the documentation has:

> MicroStack has been tested on a physical machine running either Ubuntu 18.04 LTS or Ubuntu 20.04 LTS.

**Comment by coreycb** [comment]
Thanks Peter!
